src/core/apim_payment_transactions_gateway.tf

```diff
@@ -1,249 +0,0 @@
-#######################################
-locals {
-  apim_payment_transactions_gateway_update_api = {
-    # params for all api versions
-    display_name          = "Payment Transactions Gateway - payment authorization status update"
-    description           = "RESTful APIs provided to enable payment authorization status update"
-    path                  = "payment-transactions-gateway/updates"
-    subscription_required = true
-    service_url           = null
-  }
-}
-
-resource "azurerm_api_management_api_version_set" "payment_transactions_gateway_update_api" {
-
-  name                = format("%s-payment-transactions-gateway-update-api", local.project)
-  resource_group_name = data.azurerm_resource_group.rg_api.name
-  api_management_name = data.azurerm_api_management.apim_migrated[0].name
-  display_name        = local.apim_payment_transactions_gateway_update_api.display_name
-  versioning_scheme   = "Segment"
-}
-
-module "apim_payment_transactions_gateway_update_api_v1" {
-
-  source = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v1.0.90"
-
-  name                  = format("%s-payment-transactions-gateway-update-api", local.project)
-  api_management_name   = data.azurerm_api_management.apim_migrated[0].name
-  resource_group_name   = data.azurerm_resource_group.rg_api.name
-  product_ids           = [module.apim_payment_transactions_gateway_product.product_id]
-  subscription_required = local.apim_payment_transactions_gateway_update_api.subscription_required
-  version_set_id        = azurerm_api_management_api_version_set.payment_transactions_gateway_update_api.id
-  api_version           = "v1"
-  service_url           = local.apim_payment_transactions_gateway_update_api.service_url
-
-  description  = local.apim_payment_transactions_gateway_update_api.description
-  display_name = local.apim_payment_transactions_gateway_update_api.display_name
-  path         = local.apim_payment_transactions_gateway_update_api.path
-  protocols    = ["https"]
-
-  content_format = "openapi"
-  content_value = templatefile("./api/payment_transactions_gateway_api/update/v1/_openapi.json.tpl", {
-    host = local.api_domain
-  })
-
-  xml_content = file("./api/payment_transactions_gateway_api/update/v1/_base_policy.xml.tpl")
-}
-
-################################################
-## API payment-transactions-gateway-internal  ##
-################################################
-locals {
-  apim_payment_transactions_gateway_internal_api = {
-    # params for all api versions
-    display_name          = "Payment Transactions Gateway - payment-transactions-gateway"
-    description           = "RESTful APIs provided to support payments with payment gateways"
-    path                  = "payment-transactions-gateway/internal"
-    subscription_required = true
-    service_url           = null
-  }
-}
-
-resource "azurerm_api_management_api_version_set" "payment_transactions_gateway_internal_api" {
```

Comment:
this resource expose API for bancomatpay flows invoked by the IO app, have you verified no traffic on this api?

Reply:
Given that there is a planned forced update on the IO app and this API specifically is currently getting less than 100 requests/day (gathered from APIM data pertaining last week), so I assume we are safe to remove this.
If we're not I'll remove it from this PR